flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -1090,9 +1093,9 @@ static void altitude_hold_desired(ManualControlCommandData * cmd, bool flightMod
 		if (cmd->Throttle > DEADBAND_HIGH) {
 			climb_rate = expo3((cmd->Throttle - DEADBAND_HIGH) / (1.0f - DEADBAND_HIGH), altitude_hold_expo) *
 		                         altitude_hold_maxclimbrate;
-		} else if (cmd->Throttle < DEADBAND_LOW && altitude_hold_maxclimbrate > MIN_CLIMB_RATE) {
+		} else if (cmd->Throttle < DEADBAND_LOW && altitude_hold_maxdescentrate > MIN_CLIMB_RATE) {
```

Comment:
I don't know. The [commit introducing it](https://github.com/cyphyworks/TauLabs/commit/3670c0697a4139cdeae64912499b74bc504db1b3) doesn't have any information about what it's trying to accomplish. There's no justification in the relevant code either, so the best we can do is make educated guesses. :P

My suspicion is that it's related to this comment in it's PR, #1119:
> This also has another implicit change. When using this there is no longer the song and dance to enter vario mode as there was previously. It seemed like not many people liked that. If you do not want to have a vario mode then climb rate can be set to zero and it will never enter that mode.

I think that "climb rate" refers to the UAVO field formerly known as "MaxRate".